<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMemorySearch</class>
 <widget class="QDialog" name="UIMemorySearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Search Criteria</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbType"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Value:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leValue"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Start Address:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leStartAddress"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>End Address:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leEndAddress"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbbButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UIMemorySearch</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UIMemorySearch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
